src/store/mutations.js

```diff
@@ -110,6 +110,11 @@ export default {
 	expandAccount(state, accountId) {
 		state.accounts[accountId].collapsed = false
 	},
+	updateAccountSetting(state, { accountId, key, value }) {
```

Comment:
let's call it `setAccountSetting` because it also works with values that were not set before